udata/core/organization/api.py

```diff
@@ -206,6 +208,21 @@ def delete(self, org, badge_kind):
         return badges_api.remove(org, badge_kind)
 
 
+@ns.route('/<org:org>/contact/', endpoint='org_contact_points')
+class OrgContactAPI(API):
+    @api.secure
```

Comment:
Just passing by 👋 I have no context, I might be totally wrong.

From what I understand, this is not enough to secure a write endpoint, this only checks that the user is authenticated. You need to check specific permissions explicitly in the endpoint (CanEditOrganization I guess).